tests/dummy/app/templates/demos/sl-menu.hbs

```diff
@@ -194,7 +188,7 @@ export default Ember.View.extend({
 
         // need to capture TAB key with keydown event
         Ember.$( document ).on( 'keydown.menu', function( e ) {
-            if ( 9 === e.keyCode && this.get( 'controller.keyboardInUse' ) ) {
+            if ( 9 === e.keyCode &amp;&amp; this.get( 'controller.keyboardInUse' ) ) {
                 e.preventDefault();
 
```

Comment:
We can if the information trying to draw attention to can easily be identified amongst other things going on in the same file.  This also goes along with the conversation we had yesterday regarding providing better documentation, samples and examples and not wanting to use the dummy app as the illustrative source of usage.